README.md

```diff
@@ -19,3 +19,7 @@ at your option.
 Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in the
 work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any
 additional terms or conditions.
+
+## Minimum Supported Rust Version (MSRV)
+This crate is guaranteed to compile on stable Rust 1.56 and up. It *might*
+compile with older versions but that may change in any new patch release.
```

Comment:
I know this is an old PR, but the MSRV is also included in the lib.rs docstring, and it's stated as "guaranteed to compile on latest stable Rust" in that docstring.

Reply:
ah, even better. should i update this to the 2024 edition then and just state "guaranteed to compile on latest stable Rust" in the README?

Reply:
i've left the code as-is (i.e. edition 2021) and copied the MSRV policy from `lib.rs` verbatim into the README